index.html

```diff
@@ -257,14 +261,19 @@
 	    The <em>WoT Security and Privacy Guidelines</em> represent a
 	    cross-cutting building block, which should be applied to any system
 	    implementing W3C WoT. It focuses on the secure implementation and
-	    configuration of Things.
+	    configuration of Things. Privacy is important, and we discuss mitigation 
```

Comment:
The intention here was to emphasize that HOWEVER, Privacy needs to be evaluated in the context of a concrete implementatoin.

Reply:
This added sentence will be addressed in a separate PR.